src/components/textarea/bl-textarea.ts

```diff
@@ -0,0 +1,256 @@
+import {CSSResultGroup, html, LitElement, PropertyValues, TemplateResult} from "lit";
+import { customElement, property, query, state } from 'lit/decorators.js';
+import {FormControlMixin} from "@open-wc/form-control";
+import {ifDefined} from "lit/directives/if-defined.js";
+import {event, EventDispatcher} from "../../utilities/event";
+import {classMap} from "lit/directives/class-map.js";
+import { styleMap } from 'lit/directives/style-map.js';
+import {live} from "lit/directives/live.js";
+import {textAreaValidators} from "../../utilities/form-control";
+import 'element-internals-polyfill';
+import style from './bl-textarea.css';
+
+
+export type TextareaSize = 'small' | 'medium' | 'large';
+/**
+ * @tag bl-textarea
+ * @summary Baklava Textarea component
+ */
+@customElement('bl-textarea')
+export default class BlTextarea extends FormControlMixin(LitElement){
+  static get styles(): CSSResultGroup {
+    return [style];
+  }
+
+  static formControlValidators = textAreaValidators;
+
+  @query('textarea')
+  validationTarget: HTMLTextAreaElement;
+
+  /**
+   * Name of textarea
+   */
+  @property({ type: String })
+  name = '';
+
+  /**
+   * Makes textarea a mandatory field
+   */
+  @property({ type: Boolean })
+  required = false;
+
+  /**
+   * Disables the textarea
+   */
+  @property({ type: Boolean, reflect: true })
+  disabled = false;
+
+  /**
+   * Sets expandity
+   */
+  @property({ type: Boolean, reflect: true })
+  expand = false;
+
+  /**
+   * Sets max row when expand is true
+   */
+  @property({ type: Number, reflect: true, attribute:'max-row' })
+  maxRow?: number;
```

Comment:
We didn't notice this detail in ADR phase but I think we should use plural form for this (`max-rows`) to be consistent with `rows` attribute. And it makes more sense to use plural here.

Reply:
I think your suggestion is meaningful so i changed as `max-rows`